frontend/packages/console-shared/src/components/spotlight/Spotlight.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { Popper } from '@console/shared';
+import './Spotlight.scss';
+
+type SpotlightProps = {
+  selector: string;
+  interactive?: boolean;
+};
+
+const isInViewport = (element) => {
+  const rect = element.getBoundingClientRect();
+  return (
+    rect.top >= 0 &&
+    rect.left >= 0 &&
+    rect.bottom <= (window.innerHeight || document.documentElement.clientHeight) &&
+    rect.right <= (window.innerWidth || document.documentElement.clientWidth)
+  );
+};
+
+const Spotlight: React.FC<SpotlightProps> = ({ selector, interactive }) => {
+  const element = document.querySelector(selector);
+  const { height, width } = element.getBoundingClientRect();
+  const style: React.CSSProperties = {
+    height,
+    width,
+  };
+
+  const [clicked, setClicked] = React.useState(false);
+  React.useEffect(() => {
+    if (!isInViewport(element)) {
+      element.scrollIntoView();
+    }
+    if (!interactive) {
+      const handleClick = () => setClicked(true);
+      document.addEventListener('mousedown', handleClick);
+      return () => {
+        document.removeEventListener('mousedown', handleClick);
```

Comment:
```suggestion
      element.addEventListener('mousedown', handleClick);
      return () => {
        element.removeEventListener('mousedown', handleClick);
```
This was not updated...

Reply:
I'm not sure if we need to check for click event on the element, 
`element` is the component to highlight and the UXD says a click anywhere on the screen should stop the animation

Reply:
I'll check with ux on the design again

Reply:
I checked and you are right. They want it to disappear when clicking anywhere.